ide/languages.hcl/test/unit/src/org/netbeans/modules/languages/hcl/HCLIndenterTest.java

```diff
@@ -138,7 +149,8 @@ private void performNewLineIndentationTest(String code, String golden) throws Ex
 
         assertNotSame(-1, pos);
 
-        code = code.replaceAll(Pattern.quote("|"), "");
+        code = code.replaceAll(Pattern.quote("|"), "").replace('_', ' ');
```

Comment:
btw this test could use `replace()` without having to escape the `|`, same in the method below this one.

Reply:
Well, read through the Text Blocks JEP, it seems the additional replaces are not in need after all. There are a few places where I need to add `\s` to the end of a line, marking that as non-blank, spaces are expected.

Reply:
just to be clear: my comment was about `replace` vs `replaceAll`